pkg/chartsync/git.go

```diff
@@ -314,7 +332,60 @@ func (c *GitChartSync) helmReleasesForMirror(mirror string) ([]*v1.HelmRelease,
 // per namespace, per auth.
 func mirrorName(hr *v1.HelmRelease) string {
 	if hr != nil && hr.Spec.GitChartSource != nil {
-		return hr.Spec.GitURL
+		secretName := "noauth"
```

Comment:
I think this approach is ideologically correct when secrets are being used, but this also means we will do a lot more (duplicate) polling for e.g. releases without HTTPS authentication. I would propose would be to change it to something like this:

```go
if hr != nil && hr.Spec.GitChartSource != nil {
	if hr.Spec.GitChartSource.SecretRef == nil {
		return hr.Spec.GitURL
	}
	return fmt.Sprintf("%s/%s/%s", hr.GetNamespace(), hr.Spec.GitChartSource.SecretRef.Name, hr.Spec.GitURL)
}
return ""
```

In addition, the `TODO(micahel)` comment can also be removed.